samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -91,7 +91,7 @@ CamResolutionsTest.prototype = {
     var call = new Call();
     call.pc1.addStream(stream);
     call.establishConnection();
-    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 100);
+    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 1000);
```

Comment:
I have, but increasing the frequency seems to correlate to higher failure rate on Mac but since it's already bad anyhow maybe it's bad to sacrifice sample size on other platforms. Is 100ms to fast, maybe 300 or 500?